### PR TITLE
Add create user from users list

### LINKED ISSUE #27: Add option to create a new user from list page
We need a view that will create a new user account. The fields should be: email, password, first name (not required), last name (not required), staff status, superuser status.

### PR DESCRIPTION
**Pull request by KamilSupera**
# Issue 
Closes: #27

### CODE REVIEW

File: elimar/modules/users/forms.py [hunk 1]
```diff
@@ -17,3 +20,17 @@ class Meta:
...
+class UserCreateForm(ModelForm):
+    class Meta:
+        model = User
+        fields = [
+            "email",
+            "password",
+            "first_name",
+            "last_name",
+            "is_staff",
+            "is_active",
```
Comment (kaniak274) [inline elimar/modules/users/forms.py]: `is_active` should be default by default

File: elimar/templates/users/user_create.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{% block content %}
```
Comment (kaniak274) [inline elimar/templates/users/user_create.html:3]: Could we extends this template using `base.html`?

File: elimar/modules/users/forms.py (hunk 1 above)
Comment (kaniak274) [inline elimar/modules/users/forms.py]: Cannot we remove it from the form?

Comment (KamilSupera) [reply elimar/modules/users/forms.py]: O.O

**Pull request merged**